### PR TITLE
Python3 developer guidelines

### PR DESCRIPTION
**Pull request by rlmv**
# Description

Add notes about 2/3 compatibility to the contribution guidelines

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: docs/CONTRIBUTION_TEMPLATE.md [hunk 1]
```diff
@@ -30,4 +31,29 @@ At this point, you're ready to make your changes! Feel free to ask for help; eve
 3. Submit your pull request
 
 # Code review process
-The core team looks at Pull Requests on a regular basis in a weekly triage meeting. After feedback has been given we expect responses within the month. After one month, we may close the pull request if it isn't showing any activity.
\ No newline at end of file
+The core team looks at Pull Requests on a regular basis in a weekly triage meeting. After feedback has been given we expect responses within the month. After one month, we may close the pull request if it isn't showing any activity.
+
+# Supporting Python 2 and 3
```
Comment (zduey) [inline docs/CONTRIBUTION_TEMPLATE.md]: I would add a sentence for developers using Pycharm that has a linter you can turn on that will check for compatibility across whatever versions of python you indicate. That's been a life saver for me when working in library code that isn't extremely well tested.

Comment (rlmv) [reply docs/CONTRIBUTION_TEMPLATE.md]: Added. How does that look to you?

**Comment by rlmv** [comment]
Added a note about not importing shims in test modules.

File: docs/CONTRIBUTION_TEMPLATE.md (hunk 1 above)
Comment (zduey) [reply docs/CONTRIBUTION_TEMPLATE.md]: looks good

**Pull request merged**